src/d3a/d3a_core/export.py

```diff
@@ -662,3 +664,74 @@ def _plot_avg_trade_graph(self, stats, area_name, key, label):
                               y=list(graph_obj.umHours.values()),
                               name=label.lower())
         return data_obj
+
+    def plot_energy_trade_profile_hr(self, area: Area, subdir: str):
+        """
+        Wrapper for _plot_energy_profile_hr
+        """
+        new_subdir = os.path.join(subdir, area.slug)
+        self._plot_energy_profile_hr(area, new_subdir)
+        for child in area.children:
+            if child.children:
+                self.plot_energy_trade_profile_hr(child, new_subdir)
+
+    def _plot_energy_profile_hr(self, area: Area, subdir: str):
+        """
+        Plots history of energy trades plotted for each market_slot
+        """
+        barmode = "relative"
+        xtitle = 'Time'
+        ytitle = 'Energy [kWh]'
+        market_name = area.name
+        title = f'HR Energy Trade Profile of {market_name}'
```

Comment:
```suggestion
        title = f'Hourly Energy Trade Profile of {market_name}'
```

Reply:
Why hourly? It is per market slot. 
I named it HR for High resolution. But I am open for other names as well.

Reply:
Please `High resolution` as you proposed and at least avoid confusing short forms for the title as its purpose is to aid in understanding and not lead to more confusion